src/display.rs

```diff
@@ -236,7 +236,6 @@ impl Display {
     /// Currently this only works on X11 with XComposite and XDamage extensions available.
     ///
     /// Deprecated since GDK 3.16
-    #[cfg(not(feature = "gdk_3_16"))]
```

Comment:
The problem here is that this function doesn't exist anymore from 3.16. Not just deprecated.